gogs_client/entities.py

```diff
@@ -13,17 +13,24 @@ def json_get(parsed_json, key):
     return parsed_json[key]
 
 
-class GogsUser(object):
+class GogsEntity(object):
+    def __init__(self, json):
+        self.json = json
```

Comment:
Two things:
1. What is the point of this adding a parent class? I'm not necessarily opposed to it, but I don't see a clear benefit.
2. Could we change this to `self._json`, and add a `@property` for it?